src/data/faq.json

```diff
@@ -354,7 +354,10 @@
                                 "anchor": "warn_for_others",
                                 "active": false,
                                 "textblock": [
-                                    "As an organizer of an event, starting with version 2.9, you can warn for others, such as for a person who has tested positive and was attending an event but not checked in via the Corona-Warn-App (CWA), upon request by the health department.",
+                                    "<b>Updated ?????</b>",
+                                    "This feature of the Corona-Warn-App is no longer available from version 3.0 and cannot be used with the previous versions 2.9 to 2.28 after ??????, as the verification hotline (TAN hotline) will be discontinued on this date.",

```

Comment:
from version 3.0 onwards
or
in version 3.0 and later versions

Reply:
Changed in https://github.com/corona-warn-app/cwa-website/pull/3312/commits/4ffe8f2c1cf64712dae8d52554bf516c25027efa, thanks!